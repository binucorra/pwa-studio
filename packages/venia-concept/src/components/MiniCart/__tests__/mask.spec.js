import React from 'react';
import { createTestInstance } from '@magento/peregrine';

import MiniCartMask from '../mask';

jest.mock('src/components/Mask', () => ({
    __esModule: true,
    default: () => '( Shared Mask Component Here )'
}));

test('it renders the shared Mask component', () => {
    const tree = createTestInstance(<MiniCartMask />).toJSON();

    expect(tree).toMatchSnapshot();
});
