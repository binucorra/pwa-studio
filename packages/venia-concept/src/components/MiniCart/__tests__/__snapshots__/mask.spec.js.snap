// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders the shared Mask component 1`] = `"( Shared Mask Component Here )"`;
